src/app/containers/MailboxContainer.tsx

```diff
@@ -58,7 +58,17 @@ const MailboxContainer = ({
     });
 
     const searchParams = getSearchParams(location);
-
+    const conversationMode = isConversationMode(inputLabelID, mailSettings, extractSearchParameters(location));
+    const searchParameters = useMemo<SearchParameters>(() => extractSearchParameters(location), [
```

Comment:
Could be interesting to create an helper returning the search dependency from the location.